rrfs-test/IODA/yaml/prepbufr_vadwnd.yaml

```diff
@@ -0,0 +1,348 @@
+# (C) Copyright 2024 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+      obsdatain: "./bufr/2024052700.rap.t00z.prepbufr.tm00"
+
+      exports:
+        group_by_variable: prepbufrDataLvlCat
+        subsets:
+          - VADWND    # VAD (NEXRAD) WIND REPORTS
+        variables:
+          # MetaData
+          timestamp:
+            timeoffset:
+              timeOffset: "*/DHR"
+              transforms:
+                - scale: 3600
+              referenceTime: "2024-05-27T00:00:00Z"
+          timeOffset:
+            query: "*/DHR"
+          longitude:
+            query: "*/XOB"
+          latitude:
+            query: "*/YOB"
+          stationIdentification:
+            query: "*/SID"
+          stationElevation:
+            query: "*/ELV"
+            type: float
+          prepbufrReportType:
+            query: "*/TYP"
+          dumpReportType:
+            query: "*/T29"
+          prepbufrDataLvlCat:
+            query: "*/PRSLEVEL/CAT"
+
+          pressure:
+            query: "*/PRSLEVEL/P___INFO/P__EVENT{1}/POB"
+            transforms:
+              - scale: 100
+          pressureQualityMarker:
+            query: "*/PRSLEVEL/P___INFO/P__EVENT{1}/PQM"
+          pressureError:
+            query: "*/PRSLEVEL/P___INFO/P__BACKG/POE"
+            transforms:
+              - scale: 100
+          pressureTunedError:
+            query: "*/PRSLEVEL/P___INFO/P__POSTP/POETU"
+            transforms:
+              - scale: 100
+
+          heightOfObservation:
+            query: "*/PRSLEVEL/Z___INFO/Z__EVENT{1}/ZOB"
+            type: float
```

Comment:
same here

Reply:
@guoqing-noaa, yes I added float to all/most of these. I guess there was only one that I didn't add it to. I'll ahead and remove all instances. I'm having problems replicating the issue now. If the problem comes back up we can always make this small change.